appserver/tests/tck/tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java

```diff
@@ -204,7 +204,7 @@ private void configureEnvironment(Map<String, String> env) {
         env.put("AS_DEBUG", String.valueOf(cfg.isAsadminLoggingEnabled()));
 
         env.put("PATH", cfg.getJdkDirectory().getAbsolutePath() + "/bin:" + cfg.getAntDirectory().getAbsolutePath()
-            + "/bin/:/usr/bin");
+            + "/bin/:/usr/bin:/bin");
```

Comment:
For readability - could this be broken to one-PATH-element per line?

Reply:
Hmm, yeah, I could use the String.join. I will update it ...